tests/README.md

```diff
@@ -197,8 +197,8 @@ From the root directory of the project run the following commands:
 ```shell
 # In order to build and directly push, login first
 buildah login -u="someuser" -p="topsecret" quay.io
-PUSH_MANIFEST=yes bash build-scripts/build-push-containers.sh build_base
+PUSH_MANIFEST=yes bash build-scripts/build-push-containers.sh build-base
```

Comment:
Do we need to run bash from bash?

```
# In order to build and directly push, login first
$ buildah login -u="someuser" -p="topsecret" quay.io
$ PUSH_MANIFEST="yes" ./build-scripts/build-push-containers.sh build-base

```

Reply:
I'd need to make the script executable, but otherwise it works either way, I think. I don't have a preference here - should I change it?

Reply:
Yes please, it's a shell script, so it should be executable, I don't a point why to run shell script using another instance of bash, when you are already in a bash instance

Reply:
Just note that in this case the executed script will share the variables with the calling script, and executing `exit` will exit the calling script as well.

Reply:
It's not a problem, just something to be conscious about

Reply:
Yes. I think only the scripts involved in building the rpm are used in the same instance. In all other locations the scripts are rather distinct from each other.

Reply:
Yes, e.g. `build-rpm.sh` runs `build-srpm.sh` with `source` command.
What I meant is I don't think executing a script by `bash build-scripts/build-push-containers.sh` or by `./build-scripts/build-push-containers.sh` doesn't make a difference bash-process-wise...

Reply:
But I might be wrong

Reply:
https://dev.to/amrhrabdeen/difference-between-executing-a-script-and-sourcing-it-in-linux-570f

Reply:
It's not, but it looks nicer in my opinion

Reply:
From that point of view - yes, most definitely :)